### PR TITLE
Improved error descriptions

### PR DESCRIPTION
**Pull request by kypselia**

### CODE REVIEW

File: Mantle/MTLJSONAdapter.h [hunk 1]
```diff
@@ -90,6 +90,13 @@ extern const NSInteger MTLJSONAdapterErrorInvalidJSONDictionary;
 /// does not actually exist in +propertyKeys.
 extern const NSInteger MTLJSONAdapterErrorInvalidJSONMapping;
 
+/// An exception was thrown and caught.
+extern const NSInteger MTLJSONAdapterErrorExceptionThrown;
+
+/// Associated with the NSException that was caught.
+extern NSString * const MTLJSONAdapterThrownExceptionErrorKey;
+
+
```
Comment (robb) [inline Mantle/MTLJSONAdapter.h]: Can you nix this empty line?

**Comment by robb** [comment]
Thanks for the pull request :sparkling_heart: 

Just one minor nit-pick :bow:

File: Mantle/MTLJSONAdapter.h (hunk 1 above)
Comment (kypselia) [inline Mantle/MTLJSONAdapter.h]: Sure!

**Comment by robb** [comment]
THX

**Pull request merged**